packages/edit-post/src/components/layout/style.scss

```diff
@@ -155,6 +155,7 @@
 	overflow: auto;
 
 	@include break-medium() {
+		z-index: z-index(".edit-post-layout .edit-post-post-publish-panel-break-medium");
```

Comment:
In medium viewports the user menu does not overlap the normal sidebar:
![image](https://user-images.githubusercontent.com/11271197/63283980-80a47080-c2aa-11e9-8f25-98498e0cadfb.png)

But it still overlaps the publish sidebar can we also address the issue on the smaller viewports?

Reply:
Hello @jorgefilipecosta thanks for the feedback. Now regarding this, if we would apply it to smaller views as well then it's basically apply the 99998 z-index at all times. Basically removing the following:

https://github.com/WordPress/gutenberg/blob/a5d413b998aafa49df0e1b1c520af8a7ed1862d2/assets/stylesheets/_z-index.scss#L73-L75

https://github.com/WordPress/gutenberg/blob/a5d413b998aafa49df0e1b1c520af8a7ed1862d2/packages/edit-post/src/components/layout/style.scss#L158

and just editing

https://github.com/WordPress/gutenberg/blob/a5d413b998aafa49df0e1b1c520af8a7ed1862d2/assets/stylesheets/_z-index.scss#L72

to

````scss
	".edit-post-layout .edit-post-post-publish-panel": 99998,
````

**However**, there's a comment at the top of the code block specifically saying to show the sidebar at the top of #wpadminbar. 

https://github.com/WordPress/gutenberg/blob/a5d413b998aafa49df0e1b1c520af8a7ed1862d2/assets/stylesheets/_z-index.scss#L68-L69

So i'm unsure if applying the change might lead to regression. If you think there won't be then let me know and i'll apply the changes.

Reply:
Hi @donmhico, good point. I think, for now, we can merge this PR given that it addresses an issue on the desktop, without regressions in my tests.